Sprint-1/exercises/paths.js

```diff
@@ -16,3 +16,6 @@ console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
 // Create a variable to store the ext part of the variable
+
+var dir = filePath.slice(0, lastSlashIndex);
+var ext = base.slice(4);
```

Comment:
File extension can vary in length. How would you modify your code so that your code can successfully extract file extension from `package.json`, 'libc.a` and even `filename` (with no file extension)?